ganga/GangaLHCb/Lib/Applications/GaudiExec.py

```diff
@@ -564,7 +564,11 @@ def execCmd(self, cmd):
             if cmd != 'make':
                 rc, stdout, stderr = _exec_cmd(cmd_file.name, self.directory)
         else:
-            rc, stdout, stderr = _exec_cmd(cmd_file.name, self.directory)
+            if self.apptainerBuild or 'slc6' in self.platform:
+                cmd_to_run = 'apptainer exec --env "PATH=$PATH" --bind $PWD --bind /cvmfs:/cvmfs:ro /cvmfs/cernvm-prod.cern.ch/cvm4 ' + cmd_file.name
```

Comment:
I am worried about the use if this image. There is no guarantee that /cvmfs/cernvm-prod.cern.ch is mounted at a given location. Is there not an official LHCb image for centos7?

Reply:
There is but I don't think it is on CVMFS. I guess we could add it to our won `/cvmfs` location if we really needed it.